docs/user_docs/kubeblocks-for-rabbitmq/manage-rabbitmq.md

```diff
@@ -0,0 +1,852 @@
+|---------------------------------------|--------------------------------------|
+| `spec.terminationPolicy`              | It is the policy of cluster termination. The default value is `Delete`. Valid values are `DoNotTerminate`, `Halt`, `Delete`, `WipeOut`.  <p> - `DoNotTerminate` blocks deletion operation. </p><p> - `Halt` deletes workload resources such as statefulset and deployment workloads but keep PVCs. </p><p> - `Delete` is based on Halt and deletes PVCs. </p> - `WipeOut` is based on Delete and wipe out all volume snapshots and snapshot data from a backup storage location. |
+| `spec.affinity`                       | It defines a set of node affinity scheduling rules for the cluster's Pods. This field helps control the placement of Pods on nodes within the cluster.  |
+| `spec.affinity.podAntiAffinity`       | It specifies the anti-affinity level of Pods within a component. It determines how pods should spread across nodes to improve availability and performance. |
+| `spec.affinity.topologyKeys`          | It represents the key of node labels used to define the topology domain for Pod anti-affinity and Pod spread constraints.   |
+| `spec.componentSpecs`                 | It is the list of components that define the cluster components. This field allows customized configuration of each component within a cluster.   |
+| `spec.componentSpecs.componentDefRef` | It is the name of the component definition that is defined in the cluster definition and you can get the component definition names with `kubectl get clusterdefinition qdrant -o json \| jq '.spec.componentDefs[].name'`.   |
+| `spec.componentSpecs.name`            | It specifies the name of the component.     |
+| `spec.componentSpecs.disableExporter` | It defines whether the monitoring function is enabled. |
+| `spec.componentSpecs.replicas`        | It specifies the number of replicas of the component.  |
+| `spec.componentSpecs.resources`       | It specifies the resource requirements of the component.  |
+
+KubeBlocks operator watches for the `Cluster` CRD and creates the cluster and all dependent resources. You can get all the resources created by the cluster with `kubectl get all,secret,rolebinding,serviceaccount -l app.kubernetes.io/instance=mycluster -n demo`.
+
+```bash
+kubectl get all,secret,rolebinding,serviceaccount -l app.kubernetes.io/instance=mycluster -n demo
+```
+
+Run the following command to see the created RabbitMQ cluster object:
+
+```bash
+kubectl get cluster mycluster -n demo -o yaml
+```
+
+</TabItem>
+
+</Tabs>
+
+## Connect to a RabbitMQ cluster
+
+Here is an example of connecting to the RabbitMQ cluster in the demo environment on the localhost. For more connection options, refer to [Connect Database](./../connect_database/overview-of-database-connection.md) for details.
+
+<Tabs>
+
+<TabItem value="kbcli" label="kbcli" default>
+
+Run the command below to connect to the cluster.
+
+```bash
+kbcli cluster connect <clustername>  --namespace <name>
```

Comment:
RabbitMQ is managed using RabbitMQ tools and does not support direct connections. This feature can be removed.

Reply:
Removed.